opencraft/settings.py

```diff
@@ -77,7 +77,8 @@
     'pr_watch',
     'userprofile',
     'registration',
-    'backup_swift'
+    'backup_swift',
+    'community',
```

Comment:
@bradenmacdonald 

> do you filter out GitHub replies that don't mention you too? I assumed you just did that on JIRA.

Yup - or more precisely I deprioritize the threads where I'm not mentioned, in all tools. I get too many notifications, so to spend less time on it, I sometimes look at the threads where I'm not mentioned, but not within a specific timeframe, and can just skip them altogether if I need to make time.

> If we have a .yml file in the test suite, it cannot contain a part that is randomly generated. So yeah, we can add some messy hacks/mocks to make the test suite work and successfully compare to a .yml file, but we won't be able to provide the community with a link that includes randomly generated secure vars just for them.

I meant that the variables wouldn't have to be randomly generated - we could just set the initial key to make it predictable, or mock these specific fields to have a useful self-explanatory value, like "EDXAPP_VAR = 'Set me to a random value'"?

> Of course, nobody should be using our .yml file without changing the vars anyways, but it's better if we can provide an option that's secure by default.

If we provide these values publicly, will it be secure anyway?

> I don't consider any of these things a blocker though. I'm fine with changing to the approach you're suggesting. I just think it's a bit less nice :)

Which part is less nice? I might be missing something about the value of having the randomly generated values there.